source/portable/NetworkInterface/ATSAME5x/NetworkInterface.c

```diff
@@ -350,8 +413,9 @@ static void prvEMACDeferredInterruptHandlerTask( void * pvParameters )
     }
 }
 
-BaseType_t xNetworkInterfaceOutput( NetworkBufferDescriptor_t * const pxDescriptor,
-                                    BaseType_t xReleaseAfterSend )
+BaseType_t xATSAM5x_NetworkInterfaceOutput( NetworkInterface_t * pxInterface,
+                                            NetworkBufferDescriptor_t * const pxDescriptor,
+                                            BaseType_t bReleaseAfterSend )
```

Comment:
```suggestion
                                            BaseType_t xReleaseAfterSend )
```

I'd suggest to use `xReleaseAfterSend` instead of `bReleaseAfterSend` because it's `BaseType_t` not `boolean`.